cmd/systemk-volume/README.md

```diff
@@ -0,0 +1,12 @@
+# systemk-volume
+
+This is a helper binary that chowns specific paths *before* the main units start taking over. This
+helps to move as much state from systemk into systemd.
+
+Every pod gets two extra units, one that runs before and one that runs after: they will be called:
```

Comment:
It seems implicit that this is enforced, not opt-in. It would help to be more explicit about the intent.